cli/src/main.rs

```diff
@@ -53,7 +53,7 @@ enum Commands {
     Parse(Parse),
     /// Run a parser's tests
     Test(Test),
-    /// Increment the version of a grammar
+    /// Print or increment the version of a grammar
```

Comment:
Nit, `display` over `print`

```suggestion
    /// Display or increment the version of a grammar
```